### PR TITLE
Fix for getPrefixedStyleValue method

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] SVACE-609482
[Problem] Function always returns the same value
[Solution] Method has been changed. "Return" statment was
 changed to "break" loop.

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: src/js/core/util/DOM/css.js [hunk 1]
```diff
@@ -366,7 +366,7 @@
 					if (prefixedProperties.hasOwnProperty(key)) {
 						value = styles[prefixedProperties[key]];
 						if (value && value !== "none") {
-							return value;
+							break;
```
Comment (HunseopJeong) [inline src/js/core/util/DOM/css.js:369]: I don't understand your comment.
'getPrefixedStyleValue' finds until there is a valid value and returns the value.
It doesn't seem to always return the same value.

Comment (HunseopJeong) [reply src/js/core/util/DOM/css.js:369]: @method name is wrong. need to fix.
Plus, @return seems a string.

Comment (grzegorz-czajkowski) [reply src/js/core/util/DOM/css.js:369]: I also don't understand this issue but replacing return with break may not solve this SVACE issue. Maybe loop always iterates over the same key/value?

Comment (TomaszLukawskiSam) [reply src/js/core/util/DOM/css.js:369]: > I don't understand your comment.
> 'getPrefixedStyleValue' finds until there is a valid value and returns the value.
> It doesn't seem to always return the same value.

This is issue reported by SVACE. For me, the previous version of code is clear, but Sonar Linter found a problem with return.

Comment (TomaszLukawskiSam) [reply src/js/core/util/DOM/css.js:369]: > I also don't understand this issue but replacing return with break may not solve this SVACE issue. Maybe loop always iterates over the same key/value?

I installed SonarLinter localy in my VSC. The issue not occurred after this change. The `prefixedProperties` is object like this:
```
{
	webkit: "-webkit-" + value,
	moz: "-moz-" + value,
	o: "-ms-" + value,
	ms: "-o-" + value,
	normal: value
}
```
Method check in loop if value like ` "-webkit-" + value` exists in `style` object. If exists then return this value.

Comment (HunseopJeong) [reply src/js/core/util/DOM/css.js:369]: Is it okay to return the 'value' not 'break'?

Comment (grzegorz-czajkowski) [reply src/js/core/util/DOM/css.js:369]: To me, previous version was clear but behavior is not changed with proposed change and should not introduce regression. Let's satisfy SVACE :)

**Review by grzegorz-czajkowski — APPROVED** [review]
Looks good to me.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**